bundles/org.openhab.binding.nikohomecontrol/README.md

```diff
@@ -24,13 +24,29 @@ The installation only needs to be 'connected' (registered on the Niko Home Contr
 For Niko Home Control I, the binding exposes all actions from the Niko Home Control System that can be triggered from the smartphone/tablet interface, as defined in the Niko Home Control I programming software.
 For Niko Home Control II, the binding exposes all devices in the system.
 
-Supported device types are switches, dimmers and rollershutters or blinds, thermostats and energy meters (Niko Home Control II only).
+Supported device types are switches, dimmers and rollershutters or blinds, thermostats, energy meters (Niko Home Control I only) and access control (Niko Home Control II only).
 Niko Home Control alarm and notice messages are retrieved and made available in the binding.
 
 ## Supported Things
 
 The Niko Home Control Controller is represented as a bridge in the binding.
-Connected to a bridge, the Niko Home Control Binding supports alloff actions, on/off actions (e.g. for lights or groups of lights), dimmers, rollershutters or blinds, thermostats and energy meters (only Niko Home Control II).
+Connected to a bridge, the Niko Home Control Binding supports all off actions, on/off actions (e.g. for lights or groups of lights), dimmers, rollershutters or blinds, thermostats, energy meters (Niko Home Control I only) and access control devices (Niko Home Control II only).
+
+The following thing types are available in the binding:
+
+| Thing Type          | NHC I | NHC II | Description                                                                       |
+|---------------------|:-----:|:------:|-----------------------------------------------------------------------------------|
```

Comment:
Is it normal to have ":" in your different tables for this second line ?
What is the behaviour when used ?

Reply:
:--: means the content of the column will be cneter aligned. (:-- for left aligned, --: for right aligned).